src/Microsoft.Diagnostics.EventFlow.Outputs.TableStorage/project.json

```diff
@@ -16,6 +16,5 @@
     "delaySign": true
   },
   "authors": [ "Microsoft" ],
-  "summary": "Azure Table Storage output for Microsoft Diagnostics EventFlow",
```

Comment:
Yes, they are removed internally because they are causing build warnings when they are set out of the packOptions section.